cmd/influxd/upgrade/upgrade.go

```diff
@@ -238,6 +238,12 @@ func NewCommand(v *viper.Viper) *cobra.Command {
 			Flag:  "config-file",
 			Desc:  "optional: Custom InfluxDB 1.x config file path, else the default config file",
 		},
+		{
+			DestP:   &options.target.configPath,
+			Flag:    "v2-config-path",
```

Comment:
This breaks the convention for existing V2 output paths (i.e. `--bolt-path` and `--engine-path`), but we already have `--config-file` for pointing at V1 config so I didn't want to use `--config-path` to mark the V2 output.